asa/ja-jp/Content/Resources/Stylesheets/okta-main-new-nav.css

```diff
@@ -938,6 +938,10 @@ div.toolbar-main ._Skins_okta_toolbar_nav_poc.mc-component .button.next-topic-bu
 	transform: rotate(90deg);
 }
 
+.external-link-icon {
+	margin: 0 8px 6px 4px;
```

Comment:
Should we give this a few more pixels of left margin so that it's a little ways off of the end of the last character in the text? The original class had 8 pixels, and we're replacing it with 4.

Reply:
done